django-stubs/db/models/constraints.pyi

```diff
@@ -16,6 +16,7 @@ class Deferrable(Enum):
 class BaseConstraint:
     name: str
     violation_error_message: _StrOrPromise | None
+    default_violation_error_message: str = ...
```

Comment:
This won't allow lazy translations, but lazy translations are somewhat tricky.

Note code in Django:

```
from django.utils.translation import gettext_lazy as _
...
class BaseConstraint:
    default_violation_error_message = _("Constraint “%(name)s” is violated.")
```

The `_(...)` or `gettext_lazy(...)` here returns a lazy-translated string, it's not actually an instance of `str`, we call it `_StrPromise`.

Typing it as `str` would prevent this assignment.

Most contexts that accept `_StrPromise` also accept `str`, so we use the `_StrOrPromise` type there (it's just an alias for `str | _StrPromise`).

This here should use `_StrOrPromise` as well.